src/sentry/static/sentry/app/views/admin/adminLayout.tsx

```diff
@@ -1,64 +1,64 @@
+import {RouteComponentProps} from 'react-router';
 import styled from '@emotion/styled';
 
-import space from 'app/styles/space';
+import SettingsLayout from 'app/views/settings/components/settingsLayout';
 import SettingsNavigation from 'app/views/settings/components/settingsNavigation';
 
-const AdminLayout: React.FC = ({children}) => (
-  <DocumentTitle title="Sentry Admin">
-    <Page>
-      <NavWrapper>
-        <SettingsNavigation
-          stickyTop="0"
-          navigationObjects={[
-            {
-              name: 'System Status',
-              items: [
-                {path: '/manage/', index: true, title: 'Overview'},
-                {path: '/manage/buffer/', title: 'Buffer'},
-                {path: '/manage/queue/', title: 'Queue'},
-                {path: '/manage/quotas/', title: 'Quotas'},
-                {path: '/manage/status/environment/', title: 'Environment'},
-                {path: '/manage/status/packages/', title: 'Packages'},
-                {path: '/manage/status/mail/', title: 'Mail'},
-                {path: '/manage/status/warnings/', title: 'Warnings'},
-                {path: '/manage/settings/', title: 'Settings'},
-              ],
-            },
-            {
-              name: 'Manage',
-              items: [
-                {path: '/manage/organizations/', title: 'Organizations'},
-                {path: '/manage/projects/', title: 'Projects'},
-                {path: '/manage/users/', title: 'Users'},
-              ],
-            },
-          ]}
-        />
-      </NavWrapper>
-      <Content>{children}</Content>
-    </Page>
-  </DocumentTitle>
+const AdminNavigation = () => (
+  <SettingsNavigation
+    stickyTop="0"
+    navigationObjects={[
+      {
+        name: 'System Status',
+        items: [
+          {path: '/manage/', index: true, title: 'Overview'},
+          {path: '/manage/buffer/', title: 'Buffer'},
+          {path: '/manage/queue/', title: 'Queue'},
+          {path: '/manage/quotas/', title: 'Quotas'},
+          {path: '/manage/status/environment/', title: 'Environment'},
+          {path: '/manage/status/packages/', title: 'Packages'},
+          {path: '/manage/status/mail/', title: 'Mail'},
+          {path: '/manage/status/warnings/', title: 'Warnings'},
+          {path: '/manage/settings/', title: 'Settings'},
+        ],
+      },
+      {
+        name: 'Manage',
+        items: [
+          {path: '/manage/organizations/', title: 'Organizations'},
+          {path: '/manage/projects/', title: 'Projects'},
+          {path: '/manage/users/', title: 'Users'},
+        ],
+      },
+    ]}
+  />
 );
 
-const NavWrapper = styled('div')`
-  flex-shrink: 0;
-  flex-grow: 0;
-  width: ${p => p.theme.settings.sidebarWidth};
-  background: ${p => p.theme.white};
-  border-right: 1px solid ${p => p.theme.border};
-`;
+type Props = {
+  children: React.ReactNode;
+} & RouteComponentProps<{}, {}>;
+
+class AdminLayout extends React.Component<Props> {
```

Comment:
Doesn't need to be a class component.

Reply:
I think I was under the impression that class components are preferred, but I'm coming to the conclusion that React upstream is moving towards functional components instead, so I'll change this back. Thank you for the feedback.

Reply:
https://github.com/getsentry/develop/pull/293 check this out